lib/atom.dart

```diff
@@ -610,7 +610,7 @@ class Panel extends ProxyHolder {
 
   Stream<bool> get onDidChangeVisible => eventStream('onDidChangeVisible') as Stream<bool>;
   Stream<Panel> get onDidDestroy =>
-      eventStream('onDidDestroy').map((obj) => new Panel(obj));
+      eventStream('onDidDestroy').map((obj) => new Panel(obj)) as Stream<Panel>;
```

Comment:
No, just a strong mode warning reported though atom. Not sure why it wasn't failing the build though.

Hopefully these Stream casts will go away when we have generic methods.